docs/protocols/authentication-specification.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 7
+title: Authentication
+---
+
+# Authentication Specification
+
+AdCP supports multiple authentication methods with consistent principal identification across protocols.
+
+## Authentication Methods
+
+### JWT Bearer Token
+```http
+Authorization: Bearer <jwt_token>
+```
+
+JWT tokens must include AdCP claims:
+```json
+{
+  "sub": "principal_123",
+  "adcp": {
+    "principal_type": "advertiser",
+    "tenant_id": "tenant_abc",
+    "permissions": {
+      "products": ["read"],
+      "media_buys": ["read", "write"]
+    }
+  }
+}
+```
+
+### API Key
+```http
+X-API-Key: <api_key>
+```
+
+API keys are mapped to principals server-side.
+
+## Tenant Identification
```

Comment:
I don't think this should even be known externally. The tenant should have a unique url.